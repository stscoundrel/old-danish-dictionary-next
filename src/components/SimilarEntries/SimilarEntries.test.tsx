import { DictionaryEntry } from 'lib/models/dictionary'
import renderer from 'react-test-renderer'
import SimilarEntries from './index'

describe('Similar entries component', () => {
  const entries: DictionaryEntry[] = [
    {
      headword: 'Abe',
      definitions: [],
      slug: 'abe',
    },
  ]

  test('Matches the snapshot', () => {
    const tree = renderer.create(
      <SimilarEntries entries={entries} />,
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
