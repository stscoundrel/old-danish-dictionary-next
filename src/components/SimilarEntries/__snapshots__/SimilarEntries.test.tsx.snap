// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Similar entries component Matches the snapshot 1`] = `
[
  <h4>
    Similar entries:
  </h4>,
  <ul>
    <li>
      <a
        className="link"
        href="/word/abe"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        Abe (I)
      </a>
    </li>
  </ul>,
]
`;
